src/main/scala/bitboard/Fen.scala

```diff
@@ -0,0 +1,166 @@
+package chess
+package bitboard
+
+import scala.language.implicitConversions
+import bitboard.Bitboard.*
+import scala.collection.mutable.ListBuffer
+import cats.syntax.all.*
+
+given Conversion[Rank, Int] = _.value
+
+case class Fen(board: Board, state: State):
```

Comment:
is it a good idea to name this `Fen`? this concept already exists in scalachess

Reply:
No, it's not. It will need a new name, or being removed.